packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -587,17 +676,33 @@ class BackdropFilter extends SingleChildRenderObjectWidget {
   /// type for performance reasons.
   final bool enabled;
 
+  /// The backdrop key that identifies the backdrop this filter will apply to.
+  ///
+  /// The default value for the backdrop key is `null`.
+  final BackdropKey? backdropKey;
+
+  /// Whether to look up the [backdropKey] from a parent [BackdropGroup].
+  final bool useSharedKey;
+
+  BackdropKey? _getKey(BuildContext context) {
```

Comment:
Maybe it makes more sense to make this the one public way to get the backdrop key.  So this becomes public, the ivars become private.  Otherwise the implementation details for this method are public, when what they potentially want is this.

Reply:
Lets keep them both private?